### PR TITLE
py gauge charts

### PR DESCRIPTION
**Pull request by Mahdis-z**
Doc upgrade checklist:

- [ ] random seed is set if using random data
- [x] file has been moved from `unconverted/x/y.md` to `x/y.md`
- [ ] old boilerplate at top and bottom of file has been removed
- [x] Every example is independently runnable and is optimized for short line count
- [x] no more `plot()` or `iplot()`
- [x] `graph_objs` has been renamed to `graph_objects`
- [x] `fig = <something>` call is high up in each example
- [x] minimal creation of intermediate `trace` objects
- [x] liberal use of `add_trace` and `update_layout`
- [x] `fig.show()` at the end of each example
- [ ] `px` example at the top if appropriate
- [x] minimize usage of hex codes for colors in favour of those in https://github.com/plotly/plotly.py-docs/issues/14

closes https://github.com/plotly/documentation/issues/1476

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
Can you check on why the build is not passing please?

**Comment by Mahdis-z** [comment]
> Can you check on why the build is not passing please?

yeah, checking the log

**Comment by nicolaskruchten** [comment]
@Mahdis-z can you rebase this onto master please?

File: python/bullet-charts.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    display_as: financial
+    has_thumbnail: true
+    ipynb: ~notebook_demo/11
+    language: python
+    layout: user-guide
+    name: Bullet Charts
+    order: 7
+    page_type: u-guide
+    permalink: python/bullet-charts/
+    thumbnail: thumbnail/bullet.jpg
```
Comment (nicolaskruchten) [inline python/bullet-charts.md]: this doesn't look like the same one in the JS docs... ?

File: python/bullet-charts.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        'steps': [
+            {'range': [0, 150], 'color': "gray"},
+            {'range': [150, 250], 'color': "lightgray"}],
+        'bar': {'color': "black"}}))
+
+fig.show()
+```
+
+#### Reference
+See https://plot.ly/python/reference/ for more information and chart attribute options!
```
Comment (nicolaskruchten) [inline python/bullet-charts.md]: here you should always link directly to the trace docs, so you need to link to `https://plot.ly/python/reference/#indicator`

File: python/gauge-charts.md [hunk 3]
```diff
@@ -40,177 +40,78 @@ jupyter:
...
+            'value': 490}}))
+
+fig.update_layout(paper_bgcolor = "lavender", font = {'color': "darkblue", 'family': "Arial"})
 
 fig.show()
 ```
 
+
 #### Reference
 See https://plot.ly/python/reference/ for more information and chart attribute options!
```
Comment (nicolaskruchten) [inline python/gauge-charts.md]: here you should always link directly to the trace docs, so you need to link to `https://plot.ly/python/reference/#indicator`

File: python/indicator.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    value = 450, 
+    title = {"text": "Accounts<br><span style='font-size:0.8em;color:gray'>Subtitle</span><br><span style='font-size:0.8em;color:gray'>Subsubtitle</span>"},
+    delta = {'reference': 400, 'relative': True}, 
+    domain = {'x': [0.6, 1], 'y': [0, 1]}))
+
+fig.show()
+```
+
+#### Reference
+See https://plot.ly/python/reference/ for more information and chart attribute options!
```
Comment (nicolaskruchten) [inline python/indicator.md]: here you should always link directly to the trace docs, so you need to link to `https://plot.ly/python/reference/#indicator`

File: python/bullet-charts.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+fig = go.Figure(go.Indicator(
+    mode = "number+gauge+delta", 
+    gauge = {'shape': "bullet"}, 
+    value = 220,
+    delta = {'reference': 300}, 
+    domain = {'x': [0, 1], 'y': [0, 1]},
+    title = {'text': "Profit"}))
+
+fig.show()
```
Comment (nicolaskruchten) [inline python/bullet-charts.md]: this one is too tall... can you restrict the height please?

File: python/bullet-charts.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+        'axis': {'range': [None, 300]}, 
+        'threshold': {
+            'line': {'color': "red", 'width': 2}, 
+            'thickness': 0.75, 
+            'value': 280},
+        'steps': [
+            {'range': [0, 150], 'color': "lightgray"}, 
+            {'range': [150, 250], 'color': "gray"}]}))
+
+fig.show()
```
Comment (nicolaskruchten) [inline python/bullet-charts.md]: this one is too tall... can you restrict the height please?

File: python/bullet-charts.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+        'threshold': {
+            'line': {'color': "red", 'width': 2}, 
+            'thickness': 0.75, 'value': 270},
+        'bgcolor': "white", 
+        'steps': [
+            {'range': [0, 150], 'color': "cyan"},
+            {'range': [150, 250], 'color': "royalblue"}],
+        'bar': {'color': "darkblue"}}))
+
+fig.show()
```
Comment (nicolaskruchten) [inline python/bullet-charts.md]: this one is too tall... can you restrict the height please?

**Comment by nicolaskruchten** [comment]
💃

**Pull request merged**